tests/Doctrine/Tests/Common/Annotations/AnnotationReaderTest.php

```
@@ -108,4 +110,23 @@ public function testPropertyAnnotationIsIgnored()
 
         self::assertEmpty($reader->getPropertyAnnotations($ref->getProperty('property')));
     }
+
+    /**
+     * @group gnat
+     *
+     * @runInSeparateProcess
+     */
+    public function testClassWithFullPathUseStatement()
+    {
+        try {
+            $reader = $this->getReader();
+            $ref = new \ReflectionClass(ClassWithFullPathUseStatement::class);
+
+            $annotations = $reader->getClassAnnotations($ref);
+        } catch(AnnotationException $exception) {
+            $this->fail('Class with full path use statement unable to load annotation');
```

If you don't expect an exception, let it bubble up